qa/L0_sequence_batcher/sequence_batcher_test.py

```diff
@@ -2310,6 +2316,281 @@ def test_queue_delay_full_min_util(self):
                     self.cleanup_shm_regions(precreated_shm0_handles)
                     self.cleanup_shm_regions(precreated_shm1_handles)
 
+    def test_simple_sequence_bool(self):
+        # Send one sequence and check for correct result.
+        # The result should be returned immediately.
+        for trial in _bool_trials:
+            # Run on different protocols.
+            for idx, protocol in enumerate(_protocols):
+                self.clear_deferred_exceptions()
+                try:
+                    dtype = np.bool
```

Comment:
You shouldn't need to duplicate the test for bool, what you can do is to have a list of data type to be tested, and then add a loop to iterate the types for all tests.